packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/Choices.js

```diff
@@ -1,25 +1,44 @@
+import { ResponsiveContext } from 'grommet'
 import { observer } from 'mobx-react'
 import PropTypes from 'prop-types'
-import { useState } from 'react';
-import styled, { css, withTheme } from 'styled-components'
+import { useContext, useState } from 'react';
+import styled, { withTheme } from 'styled-components'
```

Comment:
You're not using the `theme` prop for anything inside the component code, so okay to delete `withTheme` from this file and `theme` from PropTypes.